tests/bearlib/languages/LanguageMembersTypeTests.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+
+from coalib.bearlib.languages.Language import Language
+from tests.TestUtilities import LANGUAGE_NAMES
+
+
+class LanguageDefinitionTest(unittest.TestCase):
+    '''Tests for data types of LanguageDefinitionTest'''
+
+    def test_comment_delimiters_type(self):
+        for language_name in LANGUAGE_NAMES:
+            version = Language[language_name].get_default_version()
+            language = Language[version]
+            attribute = language.attributes
+            if 'comment_delimiters' in attribute:
+                assert isinstance(language.comment_delimiters, tuple)
```

Comment:
use self.assertEqual .  There is no benefit in using `assert` as pytest understands self.assertEqual